src/Jobs/RevisionsKeepLimit.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace A17\Twill\Jobs;
+
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Support\Facades\DB;
+use A17\Twill\Models\Model;
+use Exception;
+
+class RevisionsKeepLimit implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable;
+
+    protected $modelClass;
+    protected $id = 0;
+    protected $maxAttempts = 3;
+
+    public function __construct(Model $model, $id)
+    {
+        $this->modelClass = get_class($model);
+        $this->id = $id;
+    }
+
+    public function handle()
+    {
+
+        // Check if revision limits are enabled
+        if($this->checkIfLimitEnabled()){
```

Comment:
I think you can remove some of your inline comments. The method names are often descriptive enough of what is happening.